NEWS.md

```diff
@@ -1,5 +1,6 @@
 # codemetar (development version)
 
+* Fix for detecting rOpenSci review badge (#236)
```

Comment:
your GitHub handle is needed here :wink:

Reply:
doing that